### DISCUSSION TITLE
SEAT / CUPRA Login

### DISCUSSION THREAD

**Discussion by legospieler** [tillsteinbach/VWsFriend#247] [Q&A]
Is it possible to Login to Seat.ID instead of VW.Id?
I would try vwfriends with Cupra Formentor (MQB).

**Comment by tillsteinbach** [comment]
So far I did not manage to get similar data for MQB cars of Cupra. Only for Cupra Born, but unfortunately it is not compatible out of the box

**Reply by legospieler** [reply]
Last month, Seat/Cupra deactivated the web frontend and also forced users to update the (MQB)app in order to continue using the services.
Release Notes:
"Die neue Datenbank wird weiterhin Service Scheduling nutzen."
(German: The new database will continue to use service scheduling.)

Maybe you can try it again if it doesn't cause too much trouble.

Maybe they updated to same API as VW.
MIB-Updates have only been rolled out months after VW (Dec vs May)

**Reply by tillsteinbach** [reply]
I could try but I need a user willing to share his account

**Reply by legospieler** [reply]
Sharing the account should not be a problem. But my Formentor is still in production, but should be delivered to me in the next few weeks.

**Reply by daernsinstantfortress** [reply]
> I could try but I need a user willing to share his account

Not sure if your offer is still there, but I have a Born and would be willing to share my account to allow you to evaluate whether it can be added to this integration. Please reach out to me if you think this can help you.

**Reply by tillsteinbach** [reply]
I just checked two weeks ago. If I need an account to check again you will come back to
you. Thanks!

**Reply by legospieler** [reply]
Finally I got my Formentor.
So if you're still interested, I could share my account.
